docs/en/setup/backend/ttl.md

```diff
@@ -1,7 +1,7 @@
 # TTL
-In SkyWalking, besides metadata, there are two types of observability data:
-1. Record: Includes trace and alarm. Log may also be included in the future.
-1. Metric: Includes percentile, heat map, success rate, cpm (rpm), etc.
+In SkyWalking, there are two types of observability data:
```

Comment:
The summary is "two types of data" but the list items are two types of TTL, I think it's mismatch.

Reply:
Reword them.